### DISCUSSION TITLE
Simulink toolbox information

### DISCUSSION THREAD

**Discussion by SamReinsel** [mathworks/toolboxdesign#7] [Ideas]
I think something else that would be great to add would be a guide on including and sharing Simulink customization. For example

1. Custom menu items https://www.mathworks.com/help/simulink/ug/adding-items-to-model-editor-menus.html
  i. This is the "classic" custom menu options using `sl_customization` functions. Its still useful for adding custom right click menu options 
2. Custom Toolstrips https://www.mathworks.com/help/simulink/ug/create-custom-simulink-toolstrip-tabs.html
  i.  The new kid on the block: really powerful and a great evolution of 1 above for common workflows. Managing and distributing these toolstrips in a manageable way can involve a bit of work, so the best practice would be really useful.
3. Custom libraries and Adding them to the Library Browser https://www.mathworks.com/help/simulink/ug/adding-libraries-to-the-library-browser.html
  i. Custom blocks, tests for custom blocks, and doc for custom blocks is often difficult to get across to users you dont talk to regularly: even for relatively simple shared library utilities.

**Comment by rpurser47** [comment]
Thanks for the suggestion, I'm going to make this an enhancement!
